### ISSUE TITLE
Control visibility of alternate language pages

### ISSUE THREAD

**Issue by tomdickie** [Geta/SEO.Sitemaps#92]
I'm using the standard and commerce sitemaps. Language is set to * and language fallback and alternate language pages are selected. 

Is it possible to restrict the languages that show up in the alternate language pages? I want to display all of them except one.

**Comment by patkleef** [comment]
No, this is not possible at the moment. What is the use case to have this option?
